github_pr.py

```diff
@@ -149,6 +149,41 @@ def _return_specific_comment_prs(all_prs, filters):
         return []
 
 
+def _check_approved_mergers_file(mergers_file, comment_users):
+    with open(mergers_file) as f:
+        approved_users = f.read().splitlines()
+    comment_approved_users = [comment_user for comment_user in comment_users if comment_user in approved_users]
```

Comment:
I don't think you need `approved_users`. You can move this block inside the `with` block